RefrigeratorSaraFarkash/RefrigeratorSaraFarkash/CItem.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RefrigeratorSaraFarkash
+{
+    internal class CItem:IComparable<CItem>
+    {
+
+        private static int countObj = 0;
+        private int id;
+
+        public int Id
+        {
+            get { return id; }
+            //set { id = countObj; }
```

Comment:
when committing notice if you have leftover remarks

Reply:
fixed